control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -900,11 +900,12 @@ void AEB::getClosestObjectsOnPath(
     return autoware::universe_utils::createPoint(p.x, p.y, p.z);
   }();
 
+  const auto path_length = autoware::motion_utils::calcArcLength(ego_path);
   for (const auto & p : *points_belonging_to_cluster_hulls) {
     const auto obj_position = autoware::universe_utils::createPoint(p.x, p.y, p.z);
     const double obj_arc_length =
       autoware::motion_utils::calcSignedArcLength(ego_path, current_p, obj_position);
```

Comment:
@danielsanchezaran 
Is there a possibility that the starting point of ego_path is located before the ego vehicle's position?
In that case, I think there would be a difference in distance between "from current_p to ego_path endpoint" and "from ego_path start point to endpoint".

Reply:
The paths are generated from the current ego pose so the first pose is the beggining of the path, however, I forgot about the longitudinal offset between base link and front of ego, it has been fixed! @kyoichi-sugahara